exonum-java-binding/core/rust/src/runtime/cmd.rs

```diff
@@ -15,150 +15,166 @@
-
-// Parameters for `run` command
-const EJB_LOG_CONFIG_PATH: &str = "EJB_LOG_CONFIG_PATH";
-const EJB_PORT: &str = "EJB_PORT";
-const EJB_OVERRIDE_LIBRARY_PATH: &str = "EJB_OVERRIDE_LIBRARY_PATH";
-const JVM_DEBUG_SOCKET: &str = "JVM_DEBUG_SOCKET";
-const JVM_ARGS_PREPEND: &str = "JVM_ARGS_PREPEND";
-const JVM_ARGS_APPEND: &str = "JVM_ARGS_APPEND";
-// TOML keys for EJB configuration
-pub const EJB_CONFIG_SECTION_NAME: &str = "ejb";
-
-/// Encapsulates processing of extensions of the `run` command. At this step we gather additional
-/// private parameters for service configuration and optional parameters for JVM configuration and
-/// produce the complete EJB configuration that gets stored to the `Context` for further processing
-/// during the service initialization.
-pub struct Run;
-
-impl CommandExtension for Run {
-    fn args(&self) -> Vec<Argument> {
-        vec![
-            Argument::new_named(
-                EJB_PORT,
-                true,
-                "A port of the HTTP server for Java services. Must be distinct from the ports used by Exonum.",
-                None,
-                "ejb-port",
-                false
-            ),
-            Argument::new_named(
-                EJB_LOG_CONFIG_PATH,
-                false,
-                "Path to log4j configuration file.",
-                None,
-                "ejb-log-config-path",
-                false,
-            ),
-            Argument::new_named(
-                EJB_OVERRIDE_LIBRARY_PATH,
-                false,
-                "Overrides the standard path to native libraries, enabling running the non-packaged \
-                exonum-java application. Mostly for internal usage.",
-                None,
-                "ejb-override-java-library-path",
-                false,
-            ),
-            Argument::new_named(
-                JVM_DEBUG_SOCKET,
-                false,
-                "Allows JVM being remotely debugged. Takes a socket address as a parameter in form \
-                of `HOSTNAME:PORT`. For example, `localhost:8000`",
-                None,
-                "jvm-debug",
-                false,
-            ),
-            Argument::new_named(
-                JVM_ARGS_PREPEND,
-                false,
-                "Additional parameters for JVM that precede the rest of arguments. Must not have a \
-                leading dash. For example, `Xmx2G`",
-                None,
-                "jvm-args-prepend",
-                true,
-            ),
-            Argument::new_named(
-                JVM_ARGS_APPEND,
-                false,
-                "Additional parameters for JVM that get appended to the rest of arguments. Must not\
-                 have a leading dash. For example, `Xmx2G`",
-                None,
-                "jvm-args-append",
-                true,
-            ),
-        ]
+/// Exonum Java Bindings Application.
```

Comment:
Clearly not the application, but something else?

Reply:
Actually, help message of the `exonum-java` is parsed from this docstring. (see https://github.com/exonum/exonum-java-binding/pull/1157#discussion_r332459308)